online_store/gt_sports.css

```diff
@@ -0,0 +1,32 @@
+#h2title {
```

Comment:
two things here:
1. I would suggest to not use styling by id
2. this is not really a name (id or class) because it says nothing to supporter of the code. Try to understand what this title actually for and name it in more meaningful way

Reply:
Ok, I changed it into 'class', but the reason behind was, that this site was going to have only one main title anyway (thus I used 'id'). But I get it, in general, i should use 'class' when things might be repeated.